# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: content/events/src/nsEventStateManager.cpp
# Commit: 21cb609fe669
# Full Hash: 21cb609fe6690fbad6c6167e10ee2e7f9804a593
# Author: David Zbarsky <dzbarsky@gmail.com>
# Date: 2012-10-16 08:09:24
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Backout bug 779684 for topcrash
# ==============================================================================

diff -r a96deba7de41 -r 21cb609fe669 content/events/src/nsEventStateManager.cpp
--- a/content/events/src/nsEventStateManager.cpp	Mon Oct 15 01:42:40 2012 -0400
+++ b/content/events/src/nsEventStateManager.cpp	Mon Oct 15 14:35:50 2012 -0400
@@ -65,6 +65,7 @@
 #include "nsCaret.h"
 
 #include "nsSubDocumentFrame.h"
+#include "nsIFrameTraversal.h"
 #include "nsLayoutCID.h"
 #include "nsLayoutUtils.h"
 #include "nsIInterfaceRequestorUtils.h"
@@ -113,6 +114,8 @@
 
 static const nsIntPoint kInvalidRefPoint = nsIntPoint(-1,-1);
 
+static NS_DEFINE_CID(kFrameTraversalCID, NS_FRAMETRAVERSAL_CID);
+
 static bool sLeftClickOnly = true;
 static bool sKeyCausesActivation = true;
 static uint32_t sESMInstanceCount = 0;